Sources/EmbraceCore/Session/SessionSpanUtils.swift

```diff
@@ -128,3 +128,9 @@ fileprivate extension SpanPayload {
         }
     }
 }
+
+internal extension OpenTelemetryApi.Status {
+    static func sessionCrashedError() -> Status {
```

Comment:
This is not a valid status. Only 3 values are allowed (`Unset`, `Error`, and `Ok`)
https://opentelemetry.io/docs/concepts/signals/traces/#span-status

Reply:
The string passed is just an internal description, the status is still an error (it's `name` will be `error`).
From OpenTelemetryApi:

```
public enum Status: Equatable {
    /// The operation has been validated by an Application developers or Operator to have completed successfully.
    case ok
    /// The default status.
    case unset
    /// The operation contains an error.
    case error(description: String)

    /// True if this Status is OK
    public var isOk: Bool {
        return self == .ok
    }

    /// True if this Status is an Error
    public var isError: Bool {
        if case .error = self {
            return true
        }
        return false
    }

    public var name: String {
        switch self {
            case .ok:
                return "ok"
            case .unset:
                return "unset"
            case .error(description: _):
                return "error"
        }
    }

    public var code: Int {
        switch self {
            case .ok:
                return 0
            case .unset:
                return 1
            case .error(description: _):
                return 2
        }
    }
}

extension Status: CustomStringConvertible {
    public var description: String {
        if case let Status.error(description) = self {
            return "Status{statusCode=\(name), description=\(description)}"
        } else {
            return "Status{statusCode=\(name)}"
        }
    }
}
```

Reply:
👍 misread the `.error` there